backend/api.go

```diff
@@ -99,6 +99,8 @@ func (a *API) AddRoutes(router *gin.Engine) {
 	router.GET("/discounts", a.Discounts)
 	router.GET("/discounts/:id", GetToken, a.Discount)
 	router.DELETE("/discounts/:id", GetToken, a.DiscountDelete)
+
+	router.GET("/newchart", a.NewChart)
```

Comment:
Since we have multiple charts, we should make this a sub-directory like `/charts/chart2`

Reply:
If I do like `/charts/starsumchart`, will that work?

Reply:
That is fine.

Reply:
Is everything else ok? Do you see any additional problems?

Reply:
![image](https://user-images.githubusercontent.com/42430035/166866522-40b81565-f01f-4be5-a497-91227e1a0061.png)
Not sure what this error is: it's causing the reset database action to fail.

Reply:
This was actually fixed in a new commit to main. If you merge main into your branch, this should be fixed.

Reply:
Ok. I fixed it and all checks have passed.